docker/README.md

```diff
@@ -26,28 +26,43 @@ source code. The `workspace` directory should be created manually.
 
 Table of Contents
 =================
-* [Build image from development](#build-image-from-development)
+* [Build image for development](#build-image-for-development)
+* [Build image for web](#build-image-for-web)
 * [How to use docker to develop CodeCompass](#how-to-use-docker-to-develop-codecompass)
   * [Build, install and test CodeCompass](#build-install-and-test-codecompass)
   * [How to parse a project](#how-to-parse-a-project)
   * [How to start a webserver](#how-to-start-a-webserver)
+* [How to run CodeCompass webserver in docker](#how-to-run-codecompass-webserver-in-docker)
 
 ## Build image from development
 Build the development environment image. The tag name is important!
 ```
 docker build -t codecompass-dev docker/dev
 ```
 
+See more information [below](#how-to-use-docker-to-develop-codecompass) how to
+use this image to develop CodeCompass.
+
+## Build image for web
```

Comment:
When I first read this section I thought this will used the installed CodeCompass from the previous section. So I think this sections should go down above "How to run CodeCompass webserver in docker".
And in this documentation we should emphasize more that this image will contain a CodeCompass master branch built from scratch.